core/src/main/java/com/zextras/carbonio/files/dal/repositories/impl/ebean/NodeRepositoryEbean.java

```diff
@@ -578,4 +579,29 @@ public Optional<Node> getNodeByName(String nodeName, String folderId, String nod
     // resulting node and then we return the first one
     return query.findList().stream().findFirst();
   }
+
+  @Override
+  public List<Node> findNodesByOwner(String ownerId) {
+    return mDB.getEbeanDatabase().find(Node.class).where().eq(Db.Node.OWNER_ID, ownerId).findList();
+  }
+
+  @Override
+  public Optional<Node> findFirstByOwner(String ownerId) {
+    return mDB.getEbeanDatabase().find(Node.class).where().eq(Db.Node.OWNER_ID, ownerId).setMaxRows(1).findOneOrEmpty();
+  }
+
+  @Override
+  public void invertHiddenFlagNodes(List<Node> nodesToFlag) {
+    try (Transaction txn = mDB.getEbeanDatabase().beginTransaction()) {
```

Comment:
Use the `@Transactional` annotation: it automatically handles the transaction steps